package.json

```diff
@@ -76,5 +77,9 @@
     "tar": ">=4.4.18",
     "trim": ">=0.0.3",
     "trim-newlines": ">=3.0.1"
-  }
+  },
+  "browserslist": [
```

Comment:
Forgot to add this in the last PR. It tells Webpack which browser targets to build for.